app/posts/register-trainee-teachers/2025-03-29-collecting-reasons-for-deferral.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Collecting reasons for deferral
+description: Collecting reasons for deferral
+date: 2025-03-25
+tags:
+  - trainees
+  - providers
+  - deferrals
+related:
+  items:
+    - text: Understanding trainee withdrawals
+      href: /register-trainee-teachers/understanding-trainee-withdrawals/
+screenshots:
+  items:
+    - text: Outcomes for 2022-2023 academic year deferrals
+      src: 2022-2023-outcomes.png
+    - text: Outcomes for 2023-2024 academic year deferrals
+      src: 2023-2024-outcomes.png
+    - text: Deferrals free text entry interface 
+      src: Deferral-free-text-entry.png
+    - text: Check details with reason
+      src: Check-deferral-details.png
+    - text: Check details without reason
+      src: Summary-card-excluding-response.png
+---
+
+Deferring from Initial Teacher Training (ITT) often eventually leads to withdrawal, with deferred trainees much more likely to withdraw than obtain Qualified Teacher Status (QTS) as illustrated below (data sampled on March 20th 2025).
+
+<div style="display: flex; gap: 10px;">
+  <img src="2022-2023-outcomes.png" alt="Outcomes for 2022-2023 academic year deferrals" width="45%">
+  <img src="2023-2024-outcomes.png" alt="Outcomes for 2023-2024 academic year deferrals" width="45%">
+</div>
+
+With this in mind, the Department for Education sought to better understand the reasons behind trainees deferring, which would allow us to design policies and services aimed at reducing both deferrals and the number of deferrals that result in withdrawal.
```

Comment:
```suggestion
With this in mind, the Department for Education sought to understand better why trainees deferred. This research would allow us to design policies and services to reduce deferrals and the number of deferrals that result in withdrawal.
```